card/mod/edit/set/all/edit_inline.rb

```diff
@@ -1,11 +1,11 @@
 format :html do
-  view :edit_in_place, perms: :update, tags: :unknown_ok, cache: :never, wrap: :slot do
+  view :edit_inline, perms: :update, tags: :unknown_ok, cache: :never, wrap: :slot do
```

Comment:
for another time:
1. we may want to wrap this in a frame or find another way to make sure the title is visible.
2. this should have a bridge link.

will make a story...